gimme_aws_creds/okta.py

```diff
@@ -603,13 +645,14 @@ def _check_push_result(self, state_token, login_data):
 
         response_data = response.json()
 
+        # We could check for the existence of every layer...
+        # Or we can just catch AttributeError and pass, with a nosec comment.
         try:
-            if '_embedded' in response_data['_embedded']['factor']:
-                if response_data['_embedded']['factor']['_embedded']['challenge']['correctAnswer']:
-                    if self._print_correct_answer:
-                        self._print_correct_answer(response_data['_embedded']['factor']['_embedded']['challenge']['correctAnswer'])
-                        self._print_correct_answer = None
-        except:
+            if response_data['_embedded']['factor']['_embedded']['challenge']['correctAnswer']:
+                if self._print_correct_answer:
+                    self._print_correct_answer(response_data['_embedded']['factor']['_embedded']['challenge']['correctAnswer'])
+                    self._print_correct_answer = None
+        except AttributeError:  # nosec
```

Comment:
@epierce 

So, this was an easy fix.

But that made me test --action-setup-fido-authenticator...

I now have an [issue open with python-fido2](https://github.com/Yubico/python-fido2/issues/162), and a [PR open on ctap-keyring-device](https://github.com/dany74q/ctap-keyring-device/pull/10).

Both are related to enhancements made while trying to get the core functionality to work again.  I have a workaround for the fido2 issue, and the ctap-keyring-device issue means that platform devices simply won't work until the PR is merged.

I will get my changes committed shortly, neither is a complete show stopper, though I expect that @dany74q will address the ctap-keyring-device issue at the same time as the 2.0.0 release, and since that release is still a blocker for this PR...